docs/src/main/tut/docs/cassandra/README.md

```diff
@@ -4,6 +4,265 @@ title: Cassandra
 permalink: /docs/cassandra
 ---
 
-# Cassandra
 
-WIP
\ No newline at end of file
+# Freestyle-Cassandra
+
+[Cassandra] atop **Freestyle** is **`frees-cassandra`**.
+
+Freestyle Cassandra is a purely functional library for interacting with Cassandra supporting both,
+[LowQueryAPI] and [StringQueryInterpolation].
+
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+  - [What’s frees-cassandra](#whats-frees-cassandra)
+  - [Installation](#installation)
+  - [About Freestyle Cassandra](#about-freestyle-cassandra)
+  - [Freatures](#freatures)
+    - [Low level API](#low-level-api)
+    - [String Interpolator](#string-interpolator)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## What’s frees-cassandra
+
+[frees-cassandra] is a library to interact with cassandra built atop Free and following the
+[Freestyle] philosophy, being macro-powered. We use Troy under the scenes for modeling purposes 
```

Comment:
Troy is an implementation detail and can be mentioned in the credits at the end.